src/action/payment.js

```diff
@@ -247,7 +247,7 @@ class PaymentAction {
       this._nav.goPayBitcoinConfirm();
     } catch (err) {
       this._notification.display({
-        msg: `Fee estimation failed: ${err.details}`,
+        msg: `Fee estimation failed: ${err}`,
```

Comment:
@halseth can we change it so that `err` is an object that has the string on a `message` attribute so that we can access it like a normal JS Error like so: `err.message`?

Reply:
I can't really set anything, it is react-native that creates this error object.

Reply:
To clarify, we're not getting an error object here. `err` is a string.

Reply:
Ok, I guess that's what React Native throws then :)